MaskRCNN/model/boxclass_head.py

```diff
@@ -20,13 +20,13 @@
 def boxclass_outputs(feature, num_classes, seed_gen, class_agnostic_regression=False):
     """
     Args:
-        feature (any shape):
+        feature: features generated from FasterRCNN head function, Num_sampled_boxes x Num_features
         num_classes(int): num_category + 1
-        class_agnostic_regression (bool): if True, regression to N x 1 x 4
+        class_agnostic_regression (bool): if True, regression to Num_sampled_boxes x 1 x 4
 
     Returns:
-        cls_logits: N x num_class classification logits
-        reg_logits: N x num_classx4 or Nx2x4 if class agnostic
+        cls_logits: Num_sampled_boxes x Num_classes classification logits
```

Comment:
I don't think there's any sampling for boxclass_outputs. I would change `Num_sampled_boxes` to `num_boxes`